video/out/opengl/hwdec_cuda.c

```diff
@@ -424,6 +424,7 @@ static int mapper_init(struct ra_hwdec_mapper *mapper)
             .d = 1,
             .format = format,
             .render_src = true,
+            .blit_dst = p_owner->is_vk,
```

Comment:
`host_mutable` would be the correct capability to set here. (Although I'm slightly worried about the fact that this will dispatch the buffer->image copy on the transfer queue, which may either improve things or make them worse. Make sure to test this well)

Reply:
It works, and is detectable faster than using `blit_dist`. Thanks.